app/controllers/admin/studies_controller.rb

```diff
@@ -78,7 +78,8 @@ def managed_update # rubocop:todo Metrics/AbcSize
       redirect_to controller: 'admin/studies', action: 'update', id: @study.id
     end
   rescue ActiveRecord::RecordInvalid => e
-    logger.warn "Failed to update attributes: #{@study.errors.map(&:to_s)}}"
+    errors = @study.errors.full_messages
+    logger.warn "Failed to update attributes: #{errors}}"
```

Comment:
Not been able to get to the bottom of this problem, but the former way throws:

>   wrong number of arguments (given 1, expected 0)
> 
>   app/controllers/admin/studies_controller.rb:81:in `map'

Seen it in other similar contexts

Reply:
I've seen times when the .map(&:to_s) syntax doesn't work, but the .map{ |thing| thing.to_s } does, maybe it's that?

Reply:
Yep, its the same bug. In this case opted to pull it out as I figured it was less likely to get reverted than an over-eager Rubocop. Perhaps I should stick a comment above to stop an over-eager developer from reverting it.

Reply:
Although I'm just noticing that I also switched to full messages, so could probably inline it again anyway.